### ISSUE TITLE
Translation problem

### ISSUE THREAD

**Issue by elementh69** [FlutterFlow/flutterflow-issues#1724]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I will do my best to describe the problem in as much detail as possible.

**Application Settings:**
- 2 languages: PL (Polish) and EN (English)
- Prepared a template for buttons, used on most pages
- I frequently use the "duplicate page" option (where buttons are already present, and I only change their names, labels, and actions)
- Labels often contain special characters in their translations, such as ąśćżźóęłń.
- using bottom sheet actions on buttons

**Problem Description:**
In short, the problem is that when translating the label of button A on page A, it simultaneously overwrites the translation of the label of button B on page B. The same problem also occurs with components.
To facilitate understanding, I recorded a video: [Video Link](https://watch.screencastify.com/v/McP9M0tvFDOKPEva10dd)

**Additional Information:**
It seems a bit as if more buttons than one share the same ID. I would like to add that this issue is not limited to just two buttons. When reviewing the entire page of translations, I noticed that about 10 buttons had the same labels in Polish.

**Temporary Solution I Use:**
When I notice such behavior on a button, I select it, cut it, paste it in the same place, and suddenly the problem stops occurring for that button, and I can translate it without worrying that another button will take its label. However, I'm not sure how long this solution will last because I have the impression that the problem returns over time.
I already did it for most button, 1 left for bug report code generating.

### Expected Behavior

- No simultaneous translation of multiple buttons.
- No overwriting of translations of button B when translating button A.

### Steps to Reproduce

- 2 languages: PL (Polish) and EN (English)
- Prepared a template for buttons, used on most pages
- I frequently use the "duplicate page" option (where buttons are already present, and I only change their names, labels, and actions)
- Labels often contain special characters in their translations, such as ąśćżźóęłń.
- using bottom sheet actions on buttons

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4klMrqy4xOj7xY17X5bcZWhiA7H181TLJEoetxc0gbfe/4P4oPP8/sblZVcMizaH9EOFKYrHkDw8rshuLtD8YpMk2qfaZFyIdUaBPPbGCRR7rSDLuWOXEnGdNXI3aB5razpRB+HPNqLlok62WqeeuyRT6Cf9qOYwx5e6fDbOY=

### Context

Translating buttons from EN to PL

### Visual documentation

https://watch.screencastify.com/v/McP9M0tvFDOKPEva10dd

### Environment

```markdown
- FlutterFlow version: current for today
- Platform: Web on MacOS
- Browser name and version: Chrome 118.0.5993.117 
- Operating system and version affected: MacOS
```

**Comment by alecov** [comment]
Having exactly the same issue as described here. Terribly annoying.

This happens not only with the secondary language, but also with the primary language. It is indeed as if some buttons share the same internal ID or something, and it cannot be changed.

Tried with "new" buttons without copy-pasting, same issue happens sometimes (that is, sometimes they don't share texts, sometimes they do).

**Comment by elementh69** [comment]
Yeah, you right, i just realised that after spending 1h to correct all button labels it happend again on another random buttons ;/
I think for now the only way to avoid that bug is disabling translation before FF team will do something about it.
I hope it will be fixed soon.

**Comment by hariprasadms** [comment]
Hi @elementh69 - Thanks for submitting the issue. I can confirm it, this is a bug.

**Comment by wenkaifan0720** [comment]
Hi. This should be a duplicate of https://github.com/FlutterFlow/flutterflow-issues/issues/1723. I will close this one now.

**Comment by elementh69** [comment]
You right @wenkaifan0720 , my issue is looking similar to that one u mentioned, but im not sure if its the same. On my video u can see that when u change the label of a button A, then label of button B is changing realtime.

**Comment by wenkaifan0720** [comment]
Yes. The two buttons' translation id are linked together due to using the same theme widget, which is a bug on our end. Our last fix didn't catch all the edge cases, which I will fix this time.

**Comment by alecov** [comment]
This is still not good. I had to recreate a ton of buttons from scratch for the bug to disappear, however occasionally this bug reappears in the following manner: in the UI editor it's OK (apparently), but when I deploy the button has a wrong label.

This is annoying and dangerous, since it happens only on deployments.
